osu.Game.Rulesets.Osu/Edit/OsuSelectionHandler.cs

```diff
@@ -320,7 +335,109 @@ private Vector2 getClampedScale(OsuHitObject[] hitObjects, Anchor reference, Vec
         /// All osu! hitobjects which can be moved/rotated/scaled.
         /// </summary>
         private OsuHitObject[] selectedMovableObjects => SelectedItems.OfType<OsuHitObject>()
-                                                                      .Where(h => !(h is Spinner))
+                                                                      .Where(h => h is not Spinner)
```

Comment:
Kinda arbitrary at this point... but we can do with it for now. I preferred the inclusive specification because it avoids future hitobjects potentially becoming mergeable but causing an issue.